src/views/game/Game.tsx

```diff
@@ -28,6 +30,9 @@ const Game = () => {
 	const { quiz, isLoadingQuiz, errorQuiz } = useAppSelector(state => state.quiz);
 	const [selectedItem, setSelectedItem] = useState<quizState | null>(null);
 	const [expandedItem, setExpandedItem] = useState<number | null>(null);
+	const [completedQuizzes, setCompletedQuizzes] = useState<number[]>([]);
```

Comment:
Pour éviter le `@ts-ignore` : 
```suggestion
	const [completedQuizzes, setCompletedQuizzes] = useState<string[]>([]);
```